bluepyefe/formats/common.py

```diff
@@ -0,0 +1,563 @@
+from neo import io
+from collections import OrderedDict
+import os
+import json
+import hashlib
+import calendar
+import re
+import six
+import logging
+
+class manageFiles:
+    @classmethod
+    def md5(cls, filename):
+        '''
+        Generate hash md5 code for the filename passed as parameter
+        '''
+        hash_md5 = hashlib.md5()
+
+        with open(filename, "rb") as f:
+            for chunk in iter(lambda: f.read(4096), b""):
+                hash_md5.update(chunk)
+        return hash_md5.hexdigest()
+
+class manageMetadata:
+    """
+    Class to read and manage information from metadata
+    """
+
+    # pattern for string substitution
+    pattern = re.compile(r"[ \\.,\/]")
+
+    @classmethod
+    def get_cell_info(cls, filename_meta):
+        """
+        Extract cell info from the metadata file
+        """
+
+        try:
+            # if a .json metadata file exists
+            with open(filename_meta) as f:
+                data = json.load(f)
+
+                if "animal_species" not in data or "animal_species" == None:
```

Comment:
There's a mistake here.
`"animal_species" == None` will always evaluate `False`.
It is same as writing:
`if "animal_species" not in data or False:`
Which can be simplified to
`if "animal_species not in data:"`

Reply:
You probably wanted to write `data["animal_species"] == None`, I think

Reply:
@lbologna, was this resolved?

Reply:
Hello @wvangeit,

yes, this was modified for all field in the dictionary.
Thanks